CONTRIBUTING.md

```diff
@@ -151,25 +151,60 @@ If changes introduced to `master` branch result in conflicts, it should be merge
 5. Any change affecting the compiler or performance-critical features in the standard library
    should be checked with benchmarks how it affects performance.
 
-### Approval Process
+### Reviews
 
-1. To be accepted, a pull request requires recent approvals from at least two core team members; if the author is a core team
-   member, this counts as one approval. Approvals only count when based on the current code version (except for minor changes like fixing a typo).
-2. When the required approvals are given and CI is satisfied, a core team member can add the pull request to the
-   current development milestone. This signals that it is scheduled to be merged soon and gives another chance for final reviews.
-3. The wait time in this state depends on the gravity of the change and the activity of the previous discussion,
-   but it should at least be a full business day after the milestone was added.
-4. Before merging, make sure the pull request is properly labeled and its title appropriately describes the change.
-4. Finally, the pull request can be merged. Use squash merge to not pollute the version history of the main branch with
-   details of the pull request process. For non-trivial changes, the merge commit should contain a short description.
+Reviews are conducted by community members to validate a contribution and ensure quality standards are met.
+Approvals from Core Team members are required for accepting a pull requests. Other community members are encouraged to do reviews as well. Leave suggestions for improvements or approve a change when it looks good to you.
 
-This process is a guideline and not a strict rule so there might be occasional exceptions within reason.
-Urgent infrastructure fixes for example can be expected to skip the line.
+1. Make sure the [formal minimum requirements](#minimum-requirements) are met, for the change itself and the PR. Cross check with the referenced issue(s).
+2. Check if CI is successful. If not, try to figure out what's wrong and add a comment about it. If a failure seems unrelated, maintainers can try to rerun the job.
+3. Leave inline comments when you want to request changes or ask for clarification. Suggestions are often understood as requirements, so make it clear if a proposal is optional or you're just asking for feedback.
+
+### Accepting a Pull Request
+
+Check list:
+
+1. At least two approvals by Core Team members; one approval if the author is a Core Team member. Only approvals based on the most recent code version count (ignoring minor changes like fixing a typo).
```

Comment:
```suggestion
1. At least two approvals by Core Team members; one approval if the author is a Core Team member theyself. Only approvals based on the most recent code version count (ignoring minor changes like fixing a typo).
```

Reply:
I don't think `theyself` is an actual word. Maybe `themself` could work.
But IMO the sentence is clear without it.